src/libs/utils.js

```diff
@@ -258,11 +265,13 @@ export const cantBeNumber = _.flow(_.toNumber, _.isNaN)
 export const convertValue = _.curry((type, value) => {
   switch (type) {
     case 'string':
-      return value.toString()
+      // known issue where toString is incorrectly flagged:
+      // eslint-disable-next-line lodash-fp/preferred-alias
+      return _.toString(value)
     case 'number':
       return _.toNumber(value)
     case 'boolean':
-      return !(_.lowerCase(value) === 'false' || _.lowerCase(value) === 'no' || value === '0')
+      return !['false', 'no', '0', ''].includes(_.lowerCase(value))
```

Comment:
TOL(?): It looks like array.protoype.includes() is not supported in IE, it seems like using array.protoype.indexof() which is supported by IE would work in this case. However, it is not as clear/quick to comprehend [].indexof(value) === -1 as it is to read ![].includes(value) so if supporting IE is not absolutely necessary then this likely wouldn't be a worthwhile swap though

Reply:
Yup, we explicitly don't support it.

Reply:
Clarification: Are we talking IE or Edge here?

TOL: "Explicitly don't support" or "don't explicitly support"? I think there's a huge difference. For example, I think we "don't explicitly support" Firefox but we still do our best. Can't we have the same stance for all modern browsers?

Reply:
@breilly2 I was speaking about Internet Explorer not supporting includes(), however I appreciate your second point and I would also like to know in general what the team stance is on cross-browser compatibility and what we are aiming to support

Reply:
(discussed in person)